AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/Converters/BaseUserBiverter.cs

```diff
@@ -1,42 +0,0 @@
-using AppSettingsManagerApi.Domain.Conversion;
-using MySqlBaseUser = AppSettingsManagerApi.Infrastructure.MySql.BaseUser;
-
-namespace AppSettingsManagerApi.Infrastructure.MySql.Converters;
-
-/// <summary>
-/// It's useful to have some standardized converters for objects that you'll be switching between a lot
-/// </summary>
-public class BaseUserBiverter : IBidirectionalConverter<Model.BaseUser, MySqlBaseUser>
-{
-    private readonly IBidirectionalConverter<Model.Setting, Setting> _settingConverter;
-
-    public BaseUserBiverter(IBidirectionalConverter<Model.Setting, Setting> settingConverter)
-    {
-        _settingConverter = settingConverter;
-    }
-
-    public MySqlBaseUser Convert(Model.BaseUser source)
-    {
-        return new MySqlBaseUser
-        {
-            UserId = source.UserId,
-            Password = source.Password,
-            Email = source.Email,
-            Settings =
-                source.Settings?.Select(_settingConverter.Convert).ToList() ?? new List<Setting>()
-        };
-    }
-
-    public Model.BaseUser Convert(MySqlBaseUser source)
-    {
-        return new Model.BaseUser
-        {
-            UserId = source.UserId,
-            Password = source.Password,
-            Email = source.Email,
-            Settings =
-                source.Settings?.Select(_settingConverter.Convert).ToList()
-                ?? new List<Model.Setting>()
-        };
-    }
-}
```

Comment:
why was this code taken out? was it redundant because of the Permissions files that were made in this PR?

Reply:
So the User will no longer directly reference Setting objects. Instead, Users will have Permissions, the Permission objects will point both to the User and to a SettingsGroup, and the SettingsGroup points to all of the different Setting versions/objects.

It's just a more efficient way of linking the tables and making everything directly dependent on Permissions